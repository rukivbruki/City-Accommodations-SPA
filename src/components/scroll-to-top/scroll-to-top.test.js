import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ScrollToTop from './scroll-to-top.jsx';

it(`ScrollToTop snapshot`, () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(<ScrollToTop/>);
  expect(tree).toMatchSnapshot();
});
