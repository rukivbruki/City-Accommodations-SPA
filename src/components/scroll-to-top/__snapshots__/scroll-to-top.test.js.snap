// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop snapshot 1`] = `
<Route>
  [Function]
</Route>
`;
